V2

drudge opened the pull request.

drudge committed:
initial v2 support

drudge committed:
update workflow

drudge committed:
add node 21

drudge committed:
Revert "add node 21"

This reverts commit dabe8b367d4b49db6202ce5c7f875288cc78dd7f.

razorsedge commented:
I get an outlet in the Home app after I updated nodejs on my Homebridge host and installed the beta.  I still need to do a generator power test.

jcapayne committed:
Only show the switch being on if generator is in use

Per https://github.com/drudge/homebridge-mobilelink/issues/17 you can't do HomeKit automations based on "inUse" only if a switch turns on or off.  This only shows the generator switch on if it's actually running

drudge committed:
Merge pull request #19 from jcapayne/v2

Issue #17 - Only show the switch being on if generator is in use
